clang/lib/CodeGen/CGCall.cpp

```diff
@@ -4681,6 +4601,87 @@ void CallArg::copyInto(CodeGenFunction &CGF, Address Addr) const {
   IsUsed = true;
 }
 
+/// Emit the actual writing-back of a writeback.
+void CodeGenFunction::EmitWriteback(CodeGenFunction &CGF,
```

Comment:
Moving this function makes it hard to tell if it's been modified at the same time.